yang/ietf-l3vpn-ntw.yang

```diff
@@ -2117,6 +2117,22 @@ module ietf-l3vpn-ntw {
                       if-feature "vpn-common:rtg-rip";
                       description
                         "Configuration specific to RIP routing.";
+                      leaf rip-version {
```

Comment:
Do we really need this given than RFC1723 is obsoleted by RFC2453? 

Whether ripv2 and/or ripng are to be be used, is driven by the address-family

Reply:
agree! removed